### PR TITLE
i18n(ar): Translate why-astro

### PR DESCRIPTION
**Pull request by wusaby-rush**
#### What kind of changes does this PR include?
- Translated content

#### Description
- Translate "why astro?" page to Arabic

### CODE REVIEW

File: src/pages/ar/concepts/why-astro.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+---
+
+استرو **اطار ويب شامل** لبناء **مواقع سريعة**، **موجهة للمحتوى**.
+
+
+لماذا تختار استرو بدلا عن أطر الويب اﻷخرى؟ فيما يلي خمسة مبادئ تصميم أساسية تشرح سبب بناء استرو، والمشاكل التي يحلها، ولماذا قد يكون استرو الخيار الأفضل لمشروعك أو فريقك.
+
+#### لأن استرو...
+
+1. [موجه للمحتوى](#content-focused): صُمِم استرو للمواقع التي تقدم محتوى أكثر.
```
Comment (delucis) [inline src/pages/ar/concepts/why-astro.mdx:16]:
```suggestion
1. [موجه للمحتوى](#موجه-للمحتوى): صُمِم استرو للمواقع التي تقدم محتوى أكثر.
```

File: src/pages/ar/concepts/why-astro.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+2. [يعمل على الخادم اولا ](#server-first): تعمل المواقع أسرع حين يُصير HTML على الخادم .
```
Comment (delucis) [inline src/pages/ar/concepts/why-astro.mdx:17]:
```suggestion
2. [يعمل على الخادم اولا ](#يعمل-على-الخادم-اولا): تعمل المواقع أسرع حين يُصير HTML على الخادم .
```

File: src/pages/ar/concepts/why-astro.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+3. [صُمِم ليكون سريعا](#fast-by-default): نعتقد استحالة بناء مواقع بطيئة في استرو.
```
Comment (delucis) [inline src/pages/ar/concepts/why-astro.mdx:18]:
```suggestion
3. [صُمِم ليكون سريعا](#صُمِم-ليكون-سريعا): نعتقد استحالة بناء مواقع بطيئة في استرو.
```

File: src/pages/ar/concepts/why-astro.mdx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+4. [سهل الاستخدام](#easy-to-use): لا يجب ان تكون خبيرا للعمل على استرو.
```
Comment (delucis) [inline src/pages/ar/concepts/why-astro.mdx:19]:
```suggestion
4. [سهل الاستخدام](#سهل-الاستخدام): لا يجب ان تكون خبيرا للعمل على استرو.
```

File: src/pages/ar/concepts/why-astro.mdx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+5. [مليء بالمزايا، ولكن مرن](#fully-featured-but-flexible): اكثر من ١٠٠ ملحق للاختيار من بينهم.
```
Comment (delucis) [inline src/pages/ar/concepts/why-astro.mdx:20]:
```suggestion
5. [مليء بالمزايا، ولكن مرن](#مليء-بالمزايا-ولكن-مرن): اكثر من ١٠٠ ملحق للاختيار من بينهم.
```

File: src/pages/ar/concepts/why-astro.mdx [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+يمكن لموقع استرو [الفتح أسرع بنسبة 40٪ مع جافا سكريبت أقل بنسبة 90٪](https://twitter.com/t3dotgg/status/1437195415439360003) من نفس الموقع الذي تم إنشاؤه باستخدام إطار الويب React الأكثر شعبية. لكن لا تسلم بكلامنا دون تحقق: شاهد اداء استرو [يذهل](https://youtu.be/2ZEMb_H-LYE?t=8163) Ryan Carniato (منشئ أطر الويب Solid.js و Marko).
+
+## سهل الاستخدام
+
+**هدف استرو هو الوصول لكل مطور ويب.** صُمِم استرو ليشعرك بأنه مألوف بغض النظر عن مستوى المهارة أو الخبرة السابقة في تطوير الويب.
+
+بدايةً تأكدنا من إمكانية استخدام أيًا من إطارات الواجهات التي تفضلها وتعرفها بالفعل. استرو يدعم React وPreact وSvelte وVue وSolid وLit وغيرهم لإنشاء مكونات واجهة جديدة في مشروع استرو.
+
+أردنا أيضا تضمين لغة رائعة لانشاء المكونات في استرو. للقيام بذلك، أنشأنا لغة الواجهات `.astro` الخاصة بنا. والمشابهة بشدة لـ HTML: وبالتالي أي كود يعمل في HTML هو بالفعل مكون Astro صالح! ولكنه أيضا يستعير بعض المميزات من لغات المكونات الأخرى مثل تعبيرات JSX في React وتحديد مجال التنسيقات (style scooping) في Svelte وVue. يسهل هذا القرب من HTML أيضا استخدام التحسين التدريجي (progressive enhancement) وأنماط إمكانية الوصول (accessibility) الشائعة دون مجهود إضافي.
```
Comment (delucis) [inline src/pages/ar/concepts/why-astro.mdx:69]: Small typo here:

```suggestion
أردنا أيضا تضمين لغة رائعة لانشاء المكونات في استرو. للقيام بذلك، أنشأنا لغة الواجهات `.astro` الخاصة بنا. والمشابهة بشدة لـ HTML: وبالتالي أي كود يعمل في HTML هو بالفعل مكون Astro صالح! ولكنه أيضا يستعير بعض المميزات من لغات المكونات الأخرى مثل تعبيرات JSX في React وتحديد مجال التنسيقات (style scoping) في Svelte وVue. يسهل هذا القرب من HTML أيضا استخدام التحسين التدريجي (progressive enhancement) وأنماط إمكانية الوصول (accessibility) الشائعة دون مجهود إضافي.
```

**Review by delucis — COMMENTED** [review]
Looks good! Thank you @wusaby-rush! Left some comments to fix the links and spotted one typo.

File: src/pages/ar/concepts/why-astro.mdx (hunk 6 above)
Comment (wusaby-rush) [reply src/pages/ar/concepts/why-astro.mdx]: Thank you, It is ready

**Review by delucis — APPROVED** [review]
Thanks for fixing those @wusaby-rush. LGTM!

**Pull request merged**